app/src/main/java/ch/epfl/sweng/erpa/LoginActivity.java

```diff
@@ -0,0 +1,90 @@
+package ch.epfl.sweng.erpa;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+import ch.epfl.sweng.erpa.model.User;
+
+public class LoginActivity extends Activity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+    }
+
+    public void login(View view) {
+        String usernameText = ((EditText) findViewById(R.id.username)).getText().toString();
+        String passwordText = ((EditText) findViewById(R.id.password)).getText().toString();
+
+        if (usernameText.isEmpty()) {
+            createPopup(getString(R.string.noNameMessage));
+            return;
+        } else if (passwordText.isEmpty()) {
+            createPopup(getString(R.string.noPassMessage));
+            return;
+        }
+
+        if (!checkLogin(usernameText, passwordText)) {
+            createPopup(getString(R.string.incorrectLogin));
+            return;
+        }
+
+        Intent intent = new Intent(this, MainActivity.class);
+        startActivity(intent);
+    }
+
+    public void continueWithoutLogin(View view) {
+        Intent intent = new Intent(this, MainActivity.class);
```

Comment:
I don't change this because I don't think it really matters where the user is coming from at this moment. The login page is a dummy one for now and some behavior may be strange